src/Ombi/ClientApp/src/app/login/login.component.scss

```
@@ -1,209 +1,506 @@
+.brand-logo {
+  max-width: 180px;
+  max-height: 80px;
+  object-fit: contain;
+  margin: 0 auto 8px;
+  display: block;
 }
 
-.full-width {
-    width: 100%;
+.brand-title {
+  font-family: 'Inter', 'Segoe UI', -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)